apps/src/templates/instructions/CommentArea.jsx

```diff
@@ -61,3 +61,5 @@ export class CommentArea extends Component {
     );
   }
 }
+
+export const UnconnectedCommentArea = CommentArea;
```

Comment:
It looks like `CommentArea` isn't react-redux connected, so we probably don't need to export this separately for testing.